ts-defs-src/runner-api/runner-api.d.ts

```diff
@@ -24,6 +24,38 @@ interface ScreenshotsOptions extends TakeScreenshotOptions {
     pathPattern?: string;
 }
 
+interface VideoOptions {
+    /**
+     * Specifies video compose files' into single file or not.
+     */
+    singleFile?: boolean;
+    /**
+     * Specifies video recoring for failing tests only or not.
+     */
+    failedOnly?: boolean;
+    /**
+     * Specifies a ffmpeg path that will be called
+     */
+    ffmpegPath?: string;
+    /**
+     * Specifies a custom pattern to compose video files' relative path and name.
+     */
+    pathPattern?: string;
+}
+
+interface VideoEncodingOptions {
+    /**
```

Comment:
Instead of leaving it as `any` or trying to specify all possible property, we can use the following trick:
```ts
interface VideoEncodingOptions {
    // NOTE: The index signature allows specifying undescribed FFMPEG options    
    [prop: string]: unknown;
    // NOTE: Known options will have correct typings
    r?: number;
    aspect?: string;
}
```